### PR TITLE
feat: rich text editor

### PR DESCRIPTION
**Pull request by in-mai-space**
# Description

[Link to Ticket](https://trello.com/c/VLSV2c58/12-rich-text-editor-dashboard-mobile)

Rich text editor for mobile and dashboard using [Tiptap](https://tiptap.dev/docs/editor/introduction) and [Tentap](https://10play.github.io/10tap-editor/docs/intro.html)

* [Mobile editor](https://drive.google.com/file/d/1Ibl1Hxgm-sSXN2TsAUa2rgxaykPEQ69t/view?usp=sharing)
* [Dashboard editor](https://drive.google.com/file/d/1xMCit7gJb5qkAQUwfcJknWv5Eb-JdK9p/view?usp=sharing)

# How Has This Been Tested?

No tests

# Checklist

- [x] I have performed a self-review of my code
- [x] I have reached out to another developer to review my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by alderwhiteford** [comment]
could you add videos of the editors to the PR? Thanks!

File: .gitignore [hunk 1]
```diff
@@ -19,7 +19,7 @@ node_modules
 backend/tests/api/__debug_*
 .env.prod
 
-frontend/mobile/ios/
+# frontend/mobile/ios/
```
Comment (garrettladley) [inline .gitignore:22]: should we add this line back in?

File: frontend/package-lock.json
Comment (garrettladley) [inline frontend/package-lock.json:1]: should this be deleted?

**Review by garrettladley — CHANGES_REQUESTED** [review]
some nits the LGMT
keep up the great work on the frontend @in-mai-space 🐐

**Pull request closed without merging**